dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/OneAgentMetadataEnricher.java

```diff
@@ -79,25 +79,16 @@ Logger getLogger() {
   }
 
   /**
-   * Get the file name of the file to which OneAgent persists its parameters.
+   * Get the file name of the file in which OneAgent provides the metadata.
    *
-   * @param fileContents A {@link Reader} object pointing at the indirection file. Will be read
-   *     using a {@link BufferedReader}
-   * @param indirectionBaseName The prefix for the OneAgent metadata file. Lines in the indirection
-   *     file without this prefix are ignored. If null is passed as this parameter,
-   *     'dt_metadata_e617c525669e072eebe3d0f08212e8f2' is used.
+   * @param fileContents A {@link Reader} object pointing at the indirection file.
    * @return The string containing the filename, with no leading or trailing whitespace.
```

Comment:
```suggestion
   * @return The string containing the filename or null if the file is empty.
```
You don't actually trim the string and even if you would it should be needless to say.